src/main/java/uk/nhs/digital/nhsconnect/lab/results/inbound/queue/InboundQueueService.java

```diff
@@ -0,0 +1,46 @@
+package uk.nhs.digital.nhsconnect.lab.results.inbound.queue;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.jms.core.JmsTemplate;
+import org.springframework.stereotype.Component;
+import uk.nhs.digital.nhsconnect.lab.results.mesh.message.InboundMeshMessage;
+import uk.nhs.digital.nhsconnect.lab.results.utils.ConversationIdService;
+import uk.nhs.digital.nhsconnect.lab.results.utils.JmsHeaders;
+import uk.nhs.digital.nhsconnect.lab.results.utils.TimestampService;
+
+@Component
+@Slf4j
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
```

Comment:
Remember the new way to do this:
```suggestion
@RequiredArgsConstructor(onConstructor_ = @Autowired)
```

Reply:
Sure, thank you!